plonecli/cli.py

```diff
@@ -28,13 +29,26 @@ def get_templates(ctx, args, incomplete):
     invoke_without_command=True
 )
 @click.option('-l', '--list-templates', 'list_templates', is_flag=True)
+@click.option('-v', '--versions', 'versions', is_flag=True)
 @click.pass_context
-def cli(context, list_templates):
+def cli(context, list_templates, versions):
     """Plone Command Line Interface (CLI)"""
     context.obj = {}
     context.obj['target_dir'] = reg.root_folder
     if list_templates:
         click.echo(reg.list_templates())
+    if versions:
+        ws = WorkingSet()
+        version_str = 'Available packages:\n'
+        bobtemplates_dist = ws.by_key['bobtemplates.plone']
+        bobtemplates_version = bobtemplates_dist.version
+        version_str += 'bobtemplates.plone : {0}\n'.format(
```

Comment:
building up the string this way is not optimal,

just pass a whole format string to clock.echo with the versions directly applied